working-groups/WORKING-GROUPS.md

```diff
@@ -391,3 +369,31 @@ Except as otherwise noted, the content of this page is licensed under the
 [Creative Commons Attribution 4.0 License](https://creativecommons.org/licenses/by/4.0/),
```

Comment:
shouldn't this notice be at the end of this page?

Reply:
Moved.